docs/developers/dashboard.md

```diff
@@ -0,0 +1,26 @@
+# Dashboard documentation
+
+The dashboard is written with vanilla JS, CSS and HTML (and love). We made the choice to not used any frameworks like Angular or Vue.js... because dashboard is a very small web site.
+
+This dashboard uses only two externals frameworks, which are Bootstrap and Socket.io.
+
+Socket.io is useful to dynamically received events from Copilot and displayed it.
```

Comment:
s/received/receive/
s/displayed it/display them/

Reply:
Done